addons/web_dashboard/static/src/xml/dashboard.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<templates xml:space="preserve">
+    <t t-name="DashboardMain">
+        <div class="o_web_dashboard">
```

Comment:
Maybe with container-fluid then ? It's just that the bootstrap grid requires a container as a parent of row. Because of that, you have to use some hacks but it's not the cleanest way.